components/TInput/TInputArray.vue

```diff
@@ -31,6 +31,9 @@ export default {
     },
   },
   watch: {
+    internalValue: {
```

Comment:
why did you add this part?
this was already defined under `value(val)` {}.
I think we have to remove this or if you want, you can remove `old` param from 2 params (val, old).